models/index.js

```diff
@@ -32,6 +33,23 @@ Idea.hasMany(Comment, {
   foreignKey: 'idea_id',
 });
 
+Idea.hasMany(User, {
+  foreignKey: 'user_id',
+  as: 'upvote'
+});
+
+User.belongsTo(Idea, {
+  foreignKey: 'user_id',
+  through : IdeaUpvote,
+  as: 'upvotes'
+});
+
+Idea.belongsTo(User, {
```

Comment:
This might not be needed.

Reply:
@MikiWolfe These three relations should instead be replace with 2 `belongsToMany` relations. See my examples for connecting `User` to `Space` through the `SpaceMember` model. You'll need a similar set up to connect `Idea` to `User` through the `IdeaUpvote` model.

I think we will also need direct relations between `Idea` and the `IdeaUpvote` table using the standard `Idea hasMany IdeaUpvote` (no through)